app/tasks/doctor.php

```diff
@@ -75,11 +75,11 @@
             Console::log('🟢 Console access limits are enabled');
         }
         
-        if(empty(App::getEnv('_APP_OPTIONS_FORCE_HTTPS', null))) {
-            Console::log('🔴 HTTP force option is disabled');
+        if('disabled' === App::getEnv('_APP_OPTIONS_FORCE_HTTPS', 'disabled') || empty(App::getEnv('_APP_OPTIONS_FORCE_HTTPS', 'disabled'))) {
```

Comment:
More robust would be `'enabled' !== App::getEnv('_APP_OPTIONS_FORCE_HTTPS', 'disabled')` and the second part not needed.
Compared to here: https://github.com/appwrite/appwrite/blob/265fd53659e3ecde54c963e79b649f521745c176/app/controllers/general.php#L103

Reply:
100%